tests/SprykerEcoTest/Zed/Adyen/Business/FacadeTest.php

```diff
@@ -162,6 +164,49 @@ public function testHandleNotification(): void
         }
     }
 
+    /**
+     * @return void
+     */
+    public function testHandleNotificationWithAuthorizeAfterCapture(): void
```

Comment:
- Please rename the method to `testHandleNotificationWithAuthorizeEventAfterCaptureEvent`.